types/audit.d.ts

```diff
@@ -66,8 +66,7 @@ declare global {
       wastedPercent?: number;
     }
 
-    /** Type returned by Audit.audit(). Only score is required.  */
-    export interface Product {
+    interface ProductBase {
```

Comment:
to make this less work for a human to parse this, may want to leave a comment here or on the derived classes about what's going on. It's a shame usabillity-wise that tsc makes us bifurcate just to get dependent properties like this.